components/ui/row.tsx

```diff
@@ -6,7 +6,7 @@ type RowProps = {
 } & HTMLAttributes<HTMLDivElement>; // Add your own props here if needed
 
 const Row: React.FC<RowProps> = ({ children, className, ...props }) => {
-  const _className = cn('flex md:flex-row md:w-full', className);
+  const _className = cn('flex lg:flex-row lg:w-full', className);
```

Comment:
Do not use `_className`. Underscore is for private or protected variables/functions. Just use it inline.